File: sqlConnect.php
<?php
include 'errorHandler.php';
$servername = "localhost"; //use localhost because file is on the server
    $username = "user";
    $password = "password";
    $dbname = "school";  //database name
// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

$sql = "SELECT * FROM students";
$result = $conn->query($sql);

while ( $tables = $result->fetch_assoc())
{
    echo "name: ".$row['name']." age: ".$row['age']." grade: ".$row['gradeLevel']."<br>";
}
$conn->close();
?>

File: sqlConnect.py
#!/usr/bin/python
import pymysql
db = pymysql.connect(host="localhost", user="user", passwd="password", db="school")
#create cursor
cursor = db.cursor()

cur = db.cursor()



#rows = cur.fetchall()
sql = "SELECT * FROM students"

cur.execute(sql)# execute SQL query using execute() method.

#fetch all data and print 
rows = cur.fetchall()

for row in rows:
    print(row)
#if (sql != null):
#    print("you fucked up")
# disconnect from server
db.close()
cursor.close()
